cfpb/cfgov-refresh
# Ask CFPB: Add ability to remove tip styling using tip button

Currently the only ways to remove tip styling in an Ask answer field are to edit the HTML directly or to delete the tip and re-enter its contents. This PR adds the ability to remove tip styling from a selection using the answer field's tip button.

## Changes

- Update `ask_cfpb_tips.js` to remove tip styling instead of adding it again if selection is part of a tip.
- Update the way tip selection is handled to try to correct for partial selections. This should help prevent things like tips inside `<p>` elements

## Testing

1. Check out branch
2. Choose an Ask answer in admin. Highlight a selection and click tip button to add tip styling. You should now be able to remove the styling by clicking the tip button again, either immediately or after selecting some or all of the tip contents.
3. If you highlight a partial selection -- leave off the last few characters of a paragraph -- the selection should (generally) expand to contain the stragglers

## Checklist
